<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="task">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Явная инициализация подстрокой</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Явная инициализация через обращение к полям</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ввод с клавиатуры</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Инициализация через указатель</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Инициализация через ссылку</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="form">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>421</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="submit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QComboBox" name="waterResistance">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>Нет</string>
     </property>
     <item>
      <property name="text">
       <string>Нет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Да</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Серийный номер:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="serialLetter">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Влагостойкость:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="detail_1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Название:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Серийная буква:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Номера деталей:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="serialNumber">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999999999.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="detail_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="detail_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="detail_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="detail_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>260</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Время:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="hours">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>44</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="minutes">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>44</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="seconds">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>44</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
